aten/src/ATen/native/xnnpack/RegisterOpContextClass.cpp

```diff
@@ -0,0 +1,113 @@
+#ifdef USE_XNNPACK
+
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/xnnpack/Convolution.h>
+#include <ATen/native/xnnpack/Linear.h>
+#include <ATen/native/xnnpack/OpContext.h>
+#include <ATen/Tensor.h>
+#include <torch/custom_class.h>
+
+namespace at {
+namespace native {
+namespace xnnpack {
+
+namespace {
+torch::jit::class_<XNNPackLinearOpContext> register_xnnpack_linear_op_context_class() {
+  static auto register_linear_op_context_class =
+      torch::jit::class_<XNNPackLinearOpContext>("XNNPackLinearOpContext")
+          .def_pickle(
+              [](const c10::intrusive_ptr<XNNPackLinearOpContext>& op_context)
+                  -> SerializationTypeLinearPrePack { // __getstate__
+                Tensor weight;
+                c10::optional<Tensor> bias;
+                std::tie(weight, bias) = op_context->unpack();
+                return std::make_tuple(
+                    std::move(weight),
+                    std::move(bias));
```

Comment:
Seems like you're unpacking and repacking the tuple.  Why not just `return op_context->unpack();`?

Reply:
Good catch. Will fix.